andela/ah-backend-athena
# #162163277 Pagination support for articles

**What does this PR do?**
This pull request delivers a feature that adds pagination support to articles

**Description of the tasks to be completed**
Reading can be a nightmare when we have so many articles, that's where pagination comes in.
Now only two articles are returned at a time for easy readability and the rest are moved 
to the next page.

**How can this be manually tested?**
Navigate to the endpoint for returning all articles ie `api/articles` and you should see the following result

<img width="1134" alt="screenshot 2018-12-19 at 19 47 40" src="https://user-images.githubusercontent.com/39732510/50235121-363fbc80-03c8-11e9-8ff5-14285f6f66ef.png">
